client.go

```diff
@@ -25,44 +25,52 @@ var clientServerConnection net.Conn
 var clientMasterConnection rpc.Server
 //Need a map from clientId->serverConnPort
 
+// client LogicalClock
+var clientLogicalClock = 0
 
 type ClientMaster int
 
 type PutArgs struct {
```

Comment:
Move these structs to shared/shared_structs.go because they will be need to be accessed in both client and master.